### ISSUE TITLE
Custom build grabs version tag from the parent project

### ISSUE THREAD

**Issue by forbesjo** [google/shaka-player#871]
I've installed shaka-player as a dependency to my app and am creating a custom build which removes modules I don't need:
```json
...
    "prebuild": "node_modules/shaka-player/build/build.py --name bc +@complete -@offline -@cast -@hls --force",
...
```

This grabs my project's git version and messes up `shaka.Player.version`. The build process should probably fall back to using `npm_version()` somehow.

**Comment by joeyparrish** [comment]
Sounds reasonable to me.  The version comes from `calculate_version()` in `build/shakaBuildHelpers.py`.  Perhaps `git_version()` could fail when there is no `.git` in the source base.

**Comment by joeyparrish** [comment]
The fix for this has been cherry-picked and will be released in v2.1.5.
